pkg/apis/openebs.io/v1alpha1/cas_keys.go

```diff
@@ -85,6 +85,12 @@ const (
 	// PredecessorBDKey is the key to fetch the predecessor BD in case of
 	// block device replacement.
 	PredecessorBDKey = "openebs.io/bd-predecessor"
+
+	//PodDisruptionBudgetKey is the key used to identify the PDB
+	PodDisruptionBudgetKey = "openebs.io/pod-disruption-budget"
+
+	// CstorpoolInstanceLabel is the key used on pool dependent resources
+	CstorpoolInstanceLabel = "cstorpoolinstance.openebs.io/name"
```

Comment:
make it as either CStorPoolInstanceLabel or cstorpoolInstanceLabel
S need to be caps if C is caps

Reply:
Done